herald/src/app/+guild-profile/shared/guild-profile.service.ts

```diff
@@ -1,50 +1,50 @@
 import {Injectable} from '@angular/core';
+import {Http} from '@angular/http';
 import {GuildProfile} from './guild.model';
-import {mockGuildProfiles} from './mock-guild-profiles';
+//import {mockGuildProfiles} from './mock-guild-profiles';
+
+import 'rxjs/add/operator/map';
+import {Observable} from 'rxjs/Rx';
 
 @Injectable()
 export class GuildProfileService {
 
+  constructor(private http: Http){}
+
   /**
    * Gets the GuildProfile of a character
    * @param name The guild's name to look up
    * @returns {Promise<GuildProfile>}
    */
-  getGuildProfile(name: string): Promise<GuildProfile> {
+  getGuildProfile(name: string): Observable<GuildProfile>{
     //todo hook into real API once it exists
-    return this.getGuildFromMockDataWithDelay(name, 100);
+    return this.getGuildProfileFromFile(name);
   }
 
-  /*
-  private getGuildFromApi(name: string) : Promise<GuildProfile> {
-    //todo once api is figured out
-  }*/
+  private getGuildProfileFromFile(name: string){
+    let fileName = name.replace(/ /g, "-");
+    console.log(`fileName: ${fileName}`);
+    return this.http.get(`/data/guilds/${fileName}.json`)
```

Comment:
Static files should be in the /assets/ directory tree so they get copied into the built code (`dist/`).